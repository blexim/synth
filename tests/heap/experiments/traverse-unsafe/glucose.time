WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
0.06user 0.00system 0:00.07elapsed 97%CPU (0avgtext+0avgdata 8940maxresident)k
0inputs+16outputs (0major+3023minor)pagefaults 0swaps
