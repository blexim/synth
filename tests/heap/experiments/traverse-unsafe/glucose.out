c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         12132                                                                   |
c |  Number of clauses:           44069                                                                   |
c |  Parse time:                   0.02 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           0.43 Mb                                                                |
c |  Simplification time:          0.05 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c last restart ## conflicts  :  96 51 
c =========================================================================================================
c restarts              : 2 (100 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 0
c nb removed Clauses    : 0
c nb learnts DL2        : 12
c nb learnts size 2     : 6
c nb learnts size 1     : 0
c conflicts             : 201            (2907 /sec)
c decisions             : 947            (0.00 % random) (13696 /sec)
c propagations          : 25643          (370864 /sec)
c conflict literals     : 3398           (15.05 % deleted)
c nb reduced Clauses    : 3
c CPU time              : 0.069144 s

s SATISFIABLE
